templates/etc-prometheus-prometheus-yml.j2

```diff
@@ -129,3 +129,8 @@ scrape_configs:
     file_sd_configs:
     - files:
       - "/etc/prometheus/targets/*.yml"
+
+  # Targets requiring custom options
+{% for item in prometheus_custom_targets %}
+  - {{ item | to_json }}
```

Comment:
`item` needs to be serialised in a YAML compatible format. Since it's part of an indented list the multiline `to_yaml` function doesn't work (the indent parameter only seems to set the nested indentation, not the top-level). `to_json` is a way to obtain one-line serialised YAML, since in general JSON is valid YAML (I think the only exception is white space after a `:` is required in YAML but not JSON)

Reply:
Understood. I assume it's clearly working, just seemed surprising.